src/Widgets/EligibilityModal/components/Schedule/Schedule.module.css

```diff
@@ -18,6 +18,12 @@
   font-size: 18px;
 }
 
+.summary {
+  background: #F7F9FB;
```

Comment:
as mentioned we want to use CSS variables

Reply:
good point, it is fixed